File: java/oop/InterfaceApp/src/com/techlabs/icrudable/VenderDb.java
package com.techlabs.icrudable;

public class VenderDb implements Icrudable {
	@Override
	public void create() {
		System.out.println("Vender's create operation");
		
	}

	@Override
	public void read() {
		System.out.println("Vender's create operation");
	}

	@Override
	public void update() {
		System.out.println("Vender's update operation");
		
	}

	@Override
	public void remove() {
		System.out.println("Vender's remove method");
		
	}

}

File: dotNet/csharp/InterfaceCrudApp/InterfaceCrudApp/VenderDb.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace InterfaceCrudApp
{
    class VenderDb : Icrudable
    {
        public void Create()
        {
            Console.WriteLine("Vender's Create operation.......");
        }

        public void Read()
        {
            Console.WriteLine("Vender's Read operation.......");
        }

        public void Remove()
        {
            Console.WriteLine("Vender's Remove operation.......");
        }

        public void Update()
        {
            Console.WriteLine("Vender's Update operation.......");
        }
    }
}
